inference/server/oasst_inference_server/admin.py

```diff
@@ -0,0 +1,20 @@
+from loguru import logger
+from oasst_inference_server import database, models
+from oasst_shared import utils as shared_utils
+
+
+async def delete_user_from_db(session: database.AsyncSession, user_id: str) -> bool:
+    """Deletes a user."""
+    logger.info(f"Deleting user {user_id}")
+    user = await session.get(models.DbUser, user_id)
+    if not user:
+        return False
```

Comment:
I think we should just raise an exception (404) on fail here instead of returning `bool`, that's more in line with the rest of the codebase